url.bs

```diff
@@ -36,9 +36,9 @@ Boilerplate: omit feedback-header, omit conformance
 
  <li><p>Standardize on the term URL. URI and IRI are just confusing. In
  practice a single algorithm is used for both so keeping them distinct is
- not helping anyone. URL also easily wins the
- <a href="http://www.googlefight.com/index.php?word1=url&amp;word2=uri">search result popularity contest</a>.
-
+ not helping anyone. URL also easily wins <a href="https://trends.google.com/trends/explore?q=url,uri">
+ search result popularity contest</a>
```

Comment:
It seems you dropped "the" and the period at the end. The original formatting of `<a>` on a new line and no newlines inside it was also correct. Otherwise the contents of the element start with a newline.